examples/abitest/client/abitest.cc

```diff
@@ -46,32 +61,117 @@ static bool run_tests(OakABITestService::Stub* stub, const std::string& include,
   }
 
   bool success = true;
+  int disabled = 0;
   for (const auto& result : response.results()) {
+    if (result.disabled()) {
+      disabled++;
+      continue;
+    }
     LOG(INFO) << "[ " << (result.success() ? " OK " : "FAIL") << " ] " << result.name();
     if (!result.success()) {
       success = false;
       LOG(INFO) << "    Details: " << result.details();
     }
   }
+  if (disabled > 0) {
+    LOG(INFO) << " YOU HAVE " << disabled << " DISABLED ABI TEST" << ((disabled > 1) ? "S" : "");
+  }
   return success;
 }
 
+bool run_grpc_tests(OakABITestService::Stub* stub, const std::string& include,
```

Comment:
Can you describe at a high level the conceptual difference between these two sets of tests, for reference? So that it is obvious for future contributors to figure out where to add new tests.